test/Test/Database/LSMTree/Class/Monoidal.hs

```diff
@@ -494,7 +494,7 @@ prop_snapshotNoChanges h ups ups' (V.fromList -> testKeys) = ioProperty $ do
 
       let name = fromMaybe (error "invalid name") $ mkSnapshotName "foo"
 
-      snapshot name hdl1
+      createSnapshot name hdl1
```

Comment:
Oops, #448 forgot to rename the variable here from `hdl` to `tbl`. I'll open a separate PR for renaming these `hdl` identifiers to `tbl`